src/wp-includes/rest-api/endpoints/class-wp-rest-abilities-list-controller.php

```diff
@@ -0,0 +1,339 @@
+	 *
+	 * @since 6.9.0
+	 *
+	 * @return array<string, mixed> Item schema data.
+	 */
+	public function get_item_schema(): array {
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'ability',
+			'type'       => 'object',
+			'properties' => array(
+				'name'          => array(
+					'description' => __( 'Unique identifier for the ability.' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'readonly'    => true,
+				),
+				'label'         => array(
+					'description' => __( 'Display label for the ability.' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'readonly'    => true,
+				),
+				'description'   => array(
+					'description' => __( 'Description of the ability.' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'category'      => array(
+					'description' => __( 'Category this ability belongs to.' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'readonly'    => true,
+				),
+				'input_schema'  => array(
+					'description' => __( 'JSON Schema for the ability input.' ),
+					'type'        => 'object',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'output_schema' => array(
+					'description' => __( 'JSON Schema for the ability output.' ),
+					'type'        => 'object',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'meta'          => array(
+					'description' => __( 'Meta information about the ability.' ),
+					'type'        => 'object',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+			),
+			'required'   => array( 'name', 'label', 'meta', 'description', 'category', 'input_schema', 'output_schema' ),
+		);
+
+		return $this->add_additional_fields_schema( $schema );
+	}
+
+	/**
+	 * Retrieves the query params for collections.
+	 *
+	 * @since 6.9.0
+	 *
+	 * @return array<string, mixed> Collection parameters.
+	 */
+	public function get_collection_params(): array {
+		return array(
+			'context'  => $this->get_context_param( array( 'default' => 'view' ) ),
+			'page'     => array(
+				'description'       => __( 'Current page of the collection.' ),
+				'type'              => 'integer',
+				'default'           => 1,
+				'sanitize_callback' => 'absint',
+				'validate_callback' => 'rest_validate_request_arg',
+				'minimum'           => 1,
+			),
+			'per_page' => array(
+				'description'       => __( 'Maximum number of items to be returned in result set.' ),
+				'type'              => 'integer',
+				'default'           => self::DEFAULT_PER_PAGE,
+				'minimum'           => 1,
+				'maximum'           => 100,
+				'sanitize_callback' => 'absint',
+				'validate_callback' => 'rest_validate_request_arg',
```

Comment:
I don't believe any of the `sanitize_callback` or `validate_callback` args are needed here because the `type` is specified as `integer`. The REST API should validate and sanitize this automatically via `rest_sanitize_value_from_schema()`.

```suggestion
				'minimum'           => 1,
			),
			'per_page' => array(
				'description'       => __( 'Maximum number of items to be returned in result set.' ),
				'type'              => 'integer',
				'default'           => self::DEFAULT_PER_PAGE,
				'minimum'           => 1,
				'maximum'           => 100,
```

Reply:
This is what I found in the docs, too. I will update.